src/engraving/types/accessibilitystyle.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_ENGRAVING_ACCESSIBILITYSTYLEPRESET_H
+#define MU_ENGRAVING_ACCESSIBILITYSTYLEPRESET_H
+
+namespace mu::engraving {
+enum class AccessibilityStylePreset {
+    DEFAULT = 0,
+    DEFAULT_EDITED,
+    MSN_16MM,
+    MSN_16MM_EDITED,
+    MSN_18MM,
+    MSN_18MM_EDITED,
+    MSN_20MM,
+    MSN_20MM_EDITED,
+    MSN_22MM,
+    MSN_22MM_EDITED,
+    MSN_25MM,
+    MSN_25MM_EDITED
```

Comment:
@cbjeukendrup, what do you think of this compared to a separate `bool isAccessbilityStyleEdited`?

Using a bool seems neater to me, but the tricky thing is adding it to the dropdown. For example, "20mm MSN (edited)" should be a separate option directly underneath "20mm MSN", and it should disappear if the user chooses any other option. I guess that might mean a separate enum just for the QML?

Reply:
Using a bool called `edited` makes sense to me. This would then be set to `true` whenever a style setting is adjusted _after_ loading a preset, and set back to `false` when (re-)loading a preset.

For QML, instead of using an extra enum (which might be cumbersome to convert to/from), you could also use a QVariantMap that contains a value of the normal enum to describe which preset is being used, plus a boolean to say whether it was edited. You would expose this to QML as a Variant (or directly as a QVariantMap), and inside QML it is just like a JavaScript object.

Reply:
Summarising the approach:
1. We keep the preset with the regular options like Default, MSN 16MM etc.
2. We add a `bool isAccessibilityPresetEdited` in the scoreaccessibilitymodel class to store the information if the current preset has been edited.
3. The value of isAccessibilityPresetEdited is toggled based on the if there is a custom style change or a preset is loaded.
4. The method `possibleScoreStylePreset()` which basically returns the QVariantList of QVariantMaps providing the standard list of dropdown options should have the following:
    1. The QVariantMap should have an addition key called `edited` denoting if this standard preset option has been edited. This can be dynamically known by checking the `isAccessibilityPresetEdited` flag as well as the we can have a member variable called the `m_currentStylePreset` to update the relevant option.
5. In the QML, based on the information from `possibleScoreStylePreset()` we can accordingly add the `“(edited)”` prefix to the option whose bool is true.

Is my understanding correct for this one?